static/js/admin.js

```diff
@@ -438,6 +438,7 @@ function _setup_page() {
             },
             success: function () {
                 var button = row.find("button.deactivate");
+                row.find('button.open-user-form').hide();
                 button.addClass("btn-warning");
```

Comment:
With due respect, i don't understand why this is incorrect. `button.something` has been used as jquery selector several times within the file.